src/tutorials/0006/04.md

```diff
@@ -0,0 +1,15 @@
+So now our Version 1 CID contains the following fields:
+
+`<multicodec><multihash>`
+
+But, if you remember from the previous lessons, Version 0 CIDs only contain the `<multihash>` part, so how can we distinguish between different versions of CIDs? You guessed it, more prefixes!
+
+![Version Prefix](tutorial-assets/T0006L04-version-prefix.jpg)
+
+So now our CID looks like the following:
+
+`<cid-version><multicodec><multihash>`
+
+The `<cid-version>` represents the version of the CID. In the future, it will increment to 2, 3, 4, etc.
```

Comment:
Please no!

```suggestion
The `<cid-version>` represents the version of the CID (0 or 1).
```

Reply:
I can't believe you don't want to invent new versions of the CID! 😂